content/00.zksync-era/50.sdk/10.js/01.ethers/05.api/20.v6/60.bridges/02.usdc-bridge.md

```diff
@@ -0,0 +1,188 @@
+---
+title: USDCBridge
+description: A class for interacting with the custom USDC bridge.
+tags: ["zksync", "bridges", "integration", "usdc"]
+---
+
+`USDCBridge` class implements [`AbstractBridge`](./01.abstract-bridge.md) class and provides methods for depositing and
+withdrawing USDC tokens using custom [USDC bridge](https://github.com/matter-labs/usdc-bridge).
+
+### `constructor(wallet: Wallet)`
+
+Creates a `USDCBridge` object.
+
+**Parameters:**
+
+| Parameter  | Type     | Description               |
+| ---------- | -------- | ------------------------- |
+| `wallet`   | `Wallet`| ZKsync wallet instance.   |
+
+**Example:**
+
+```ts
+import { Provider, types, Wallet, USDCBridge } from "zksync-ethers";
```

Comment:
It would be nice to have these examples imported from the `examples` folder using the `code-import` component. This way they are easier to maintain and make sure they compile and work